external/ilasm/ilasm.depproj

```diff
@@ -6,6 +6,22 @@
     <NugetRuntimeIdentifier>$(RuntimeOS)-x64</NugetRuntimeIdentifier>
```

Comment:
That's still getting pulled in through the build configuration, unless I'm misunderstanding the question.